bucketlist/core_app/views.py

```diff
@@ -4,6 +4,7 @@
 import datetime
```

Comment:
Thanks I have noted this. I have changed 3 files that have a different import order. 
This [commit](https://github.com/HawiCaesar/smart-goals-api/pull/10/commits/37978b60a10f2c7f5b8db245b16af96cd8659ce8) has the changes